### PR TITLE
Updated MedicationAdministration DSTU2 Resource

### PR DESCRIPTION
**Pull request by kckeim**
Many formatting updates in this one, and substantial re-wording of sections to be more descriptive and in a similar style to other resources that were recently updated.
Verified good links through document.
Validated that API calls behave as expected with both the expected and unexpected parameters.

Description
----
- Rewriting of Overview, including link over to `Immunization` Resource.
- Condensed / removed duplicate "IMPORTANT NOTE" sections throughout document.
- Removed the "NOTE" section throughout the entire document about "these corrections may be breaking changes..."
  - shouldn't be necessary at this time
- Removed a couple hidden <br/> characters
- Adjustments to Search Parameter Table and Implementation Notes sections.
  - Matched currently approved formatting for the table.
  - Noted parameters that accept comma-separated lists.
  - Rewrite of `effectivedate` section to match other recent resources.
- Uppercase'd "id" to "ID" in a couple of places.
- Removed unused links, and updated/applied existing links to HL7.org resources onto appropriate phrases.

********************************
PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [X] Screenshot(s) of changes attached after changes merged and published.
********************************
Old Overview section:
image

New Overview:
image

********************************
Old medication-specific note:
image

New, with added embedded links:
image

********************************
Old Parameter Table + Notes:
image

New Parameter Table + Notes:
image
image

********************************
Old Retrieve by ID Header:
image

New Retrieve by ID Header:
image

### CODE REVIEW

**Review by napster235 — APPROVED** [review]

File: content/millennium/dstu2/medications/medication-administration.md [hunk 1]
```diff
@@ -160,21 +157,15 @@ _Implementation Notes_
...
 [`_count`]: http://hl7.org/fhir/DSTU2/search.html#count
 [`number`]: http://hl7.org/fhir/DSTU2/search.html#number
-[`CodeableConcept`]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
+[CodeableConcept]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
 [contained]: http://hl7.org/fhir/DSTU2/references.html#contained
-[MedicationAdministration.medication]: http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
 [Infuse Over Time]: #custom-extensions
+[`Immunization`]: ../immunization/#overview
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:171]: probably don't need the anchor reference

File: content/millennium/dstu2/medications/medication-administration.md [hunk 2]
```diff
@@ -9,13 +9,11 @@ title: MedicationAdministration | DSTU 2 API
...
-* **NOTE**:  These corrections may be breaking changes for your use cases.  Current users of the MedicationAdministration API should test in their non-production domains.  Cerner will coordinate with you and your developers of 3rd party applications to enable in your production domains.
+* **IMPORTANT NOTE**: Infuse or Bolus administration results for continuous infusion orders may require additional calculations to determine the actual amount of medication administered per ingredient. An additional calculation is required when the dosage dose does not equal the [contained] Medication ingredient strength denominator. In this case, calculate the ratio between the ingredient’s numerator divided by the denominator and multiply by dosage dose. This will be the actual amount of medication administered.
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:15]: second to last sentence, should it be dosage quantity?

File: content/millennium/dstu2/medications/medication-administration.md [hunk 3]
```diff
@@ -34,6 +32,15 @@ The following fields are returned if valued:
   * [Site](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.site_x_){:target="_blank"}
   * [Rate](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.rate_x_){:target="_blank"}
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:33]: can you update these to httpses?

File: content/millennium/dstu2/medications/medication-administration.md [hunk 4]
```diff
@@ -34,6 +32,15 @@ The following fields are returned if valued:
...
 
+* MedicationAdministration.medication may be a reference to a [contained] Medication when the Medication cannot be represented by a [CodeableConcept] because it contains a unique combination of ingredients. Medications in the system always exist within the context of a MedicationAdministration and cannot be referenced independently.
+
+
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:37]: nit: I think there's an extra line here

File: content/millennium/dstu2/medications/medication-administration.md [hunk 5]
```diff
@@ -50,22 +57,16 @@ The following fields are returned if valued:
 
 All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/dstu2/StructureDefinition/{id}`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:58]: let's add a reference link to StructureDefinition

File: content/millennium/dstu2/medications/medication-administration.md [hunk 6]
```diff
@@ -160,21 +157,15 @@ _Implementation Notes_
...
 [`quantity`]: http://hl7.org/fhir/DSTU2/search.html#quantity
 [`_count`]: http://hl7.org/fhir/DSTU2/search.html#count
 [`number`]: http://hl7.org/fhir/DSTU2/search.html#number
-[`CodeableConcept`]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
+[CodeableConcept]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
 [contained]: http://hl7.org/fhir/DSTU2/references.html#contained
-[MedicationAdministration.medication]: http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_
 [errors]: ../../#client-errors
 [OperationOutcomes]: ../../#operation-outcomes
 [Infuse Over Time]: #custom-extensions
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:170]: so, this is a link only used once and it's a reference to the line below it? unless I'm missing something I do see that the list of fields mentions it, but it has its own link to the higher-level Extensions header.

File: content/millennium/dstu2/medications/medication-administration.md [hunk 7]
```diff
@@ -74,21 +75,29 @@ _Implementation Notes_
...
+`patient`       | Conditionally | [`reference`] | The patient who has received the medication administration. This parameter is required if the `_id` parameter is not used. Example: `patient=1234`
+`status`        | No            | [`token`]     | The status of the medication administration event. It may be a list separated by commas. Example: `status=completed,not-done`
+`practitioner`  | No            | [`reference`] | The identity of the clinician who administered the medication. It may be a list separated by commas. Example: `practitioner=1245,7659`
+`notgiven`      | No            | [`token`]     | Administrations that were not made. A value of `true` will search for these, and a value of `false` will exclude them. 
+`effectivetime` | No            | [`date`]      | A date range for when the administration happened. Example: `effectivetime=ge2014-05-19T20:54:02.000Z`
+[`_count`]      | No            | [`number`]    | The maximum number of results to return. Example: `50`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:86]: for consistency, change this example to `_count=50`

File: content/millennium/dstu2/medications/medication-administration.md [hunk 8]
```diff
@@ -74,21 +75,29 @@ _Implementation Notes_
...
-`notgiven`      | N                  | [`token`]     | Administrations that were not made. A value of 'true' will search for these, and a value of 'false' will exclude them. Example: `true`
-`effectivetime` | N                  | [`date`]      | The effectivetime search may be provided once with either the ge or le prefix or twice with the ge and le prefixes to indicate a specific range. Example: `effectivetime=ge2010-08-18` or `effectivetime=le2015-07-17` or `effectivetime=ge2010-08-18&effectivetime=le2015-07-17`
-[`_count`]      | N                  | [`number`]    | The maximum number of results to return.
-
+ Name           | Required?     | Type          | Description
+----------------|---------------|---------------|-----------------------------------------------------------------------------------------------
+`_id`           | Conditionally | [`token`]     | The logical resource id associated with the resource. It may be a list separated by commas. This parameter is required if the `patient` parameter is not used. Example: `_id=12345`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:80]: id -> ID

File: content/millennium/dstu2/medications/medication-administration.md [hunk 9]
```diff
@@ -74,21 +75,29 @@ _Implementation Notes_
...
+`patient`       | Conditionally | [`reference`] | The patient who has received the medication administration. This parameter is required if the `_id` parameter is not used. Example: `patient=1234`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:81]: capitalize Patient in the description?

Comment (amcginn) [reply content/millennium/dstu2/medications/medication-administration.md:81]: can remove 'has' as well

File: content/millennium/dstu2/medications/medication-administration.md [hunk 10]
```diff
@@ -74,21 +75,29 @@ _Implementation Notes_
...
+`status`        | No            | [`token`]     | The status of the medication administration event. It may be a list separated by commas. Example: `status=completed,not-done`
+`practitioner`  | No            | [`reference`] | The identity of the clinician who administered the medication. It may be a list separated by commas. Example: `practitioner=1245,7659`
+`notgiven`      | No            | [`token`]     | Administrations that were not made. A value of `true` will search for these, and a value of `false` will exclude them. 
+`effectivetime` | No            | [`date`]      | A date range for when the administration happened. Example: `effectivetime=ge2014-05-19T20:54:02.000Z`
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:85]: it's a dateTime

File: content/millennium/dstu2/medications/medication-administration.md [hunk 11]
```diff
@@ -160,21 +157,15 @@ _Implementation Notes_
...
-
 [`reference`]: http://hl7.org/fhir/DSTU2/search.html#reference
 [`token`]: http://hl7.org/fhir/DSTU2/search.html#token
 [`date`]: http://hl7.org/fhir/DSTU2/search.html#date
 [`quantity`]: http://hl7.org/fhir/DSTU2/search.html#quantity
 [`_count`]: http://hl7.org/fhir/DSTU2/search.html#count
 [`number`]: http://hl7.org/fhir/DSTU2/search.html#number
-[`CodeableConcept`]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
+[CodeableConcept]: http://hl7.org/fhir/DSTU2/datatypes.html#codeableconcept
 [contained]: http://hl7.org/fhir/DSTU2/references.html#contained
```
Comment (amcginn) [inline content/millennium/dstu2/medications/medication-administration.md:167]: httpsify!

**Review by amcginn — APPROVED** [review]

**Review by RicardoCrespo-ora — APPROVED** [review]

File: content/millennium/dstu2/medications/medication-administration.md (hunk 2 above)
Comment (kckeim) [reply content/millennium/dstu2/medications/medication-administration.md:15]: Yes.

File: content/millennium/dstu2/medications/medication-administration.md (hunk 9 above)
Comment (kckeim) [reply content/millennium/dstu2/medications/medication-administration.md:81]: Re-wording to match R4's better.  That one spoke it better anyways.

**Review by cldeitrich — APPROVED** [review]
[PRODOC-203242](https://jira.cerner.com/browse/PRODOC-203242): Approved for style.

File: content/millennium/dstu2/medications/medication-administration.md (hunk 1 above)
Comment (amcginn) [reply content/millennium/dstu2/medications/medication-administration.md:172]:
```suggestion
[`Immunization`]: ../immunization/
```

**Comment by Surya-Rnsit** [comment]
Validated after the code is published:
Screenshot 2024-02-02 at 3 46 18 PM
Screenshot 2024-02-02 at 3 46 25 PM

**Pull request merged**